### ISSUE TITLE
Dig Spell with Prosperity barely produces extra material

### ISSUE THREAD

**Issue by xenxero1** [Growlith1223/ArsMagica2#206]
My dig spell with Prosperity , and Mining level 3, held within a red spell book, rarely gives multiple drops. I have confirmed that it does occasionally give me a second piece of whatever I'm mining, but in my experience I've had about a  60% chance to get extra, and 85% that it only gives 1 extra piece. I've noticed this repeatedly with diamond ore, coal, and Certus Quartz ore from Applied Energistics 2. When held, the spell gains looting (which was stated as being non-functional atm), and fortune IV. I can get much better results mining with a fortune 3 pick, and the cost of prosperity 3 leads me to believe I should be getting quite a bit more than I am.

**Comment by ProfHugo** [comment]
yea it was "half-fixed" in #158, apparently the bootleg re-implementation doesn't work that well lul

**Comment by xenxero1** [comment]
Yeah, but when he said half fixed I thought he meant it just works for mining now, but not combat yet. Not that you only get half of the effect LOL!

**Comment by ProfHugo** [comment]
Well you get half the effect AND half the features for the price of one! Isn't that great? /s

**Comment by xenxero1** [comment]
I was thinking about it, and I realized. It seems to act a lot like fortune 1 (which I have 3 prosperity on it) with its drop rates. Could be that it's recognizing it has fortune, but not multiple levels.

**Comment by ProfHugo** [comment]
lul personally I think it's a really bad idea to rely on enchantment levels for counting loot, since it made it possible to: enchant a spell with an anvil or some sort and give a spell bonus drop without affecting mana cost and it also made it possible for the enchantments to be disenchanted.
